File: java/quiteaproblem.java
import java.util.Scanner;

public class quiteaproblem {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		while (scan.hasNextLine())
			System.out.println(scan.nextLine().toLowerCase().contains("problem") ? "yes" : "no");
		scan.close();
	}
}

File: ts/src/quiteaproblem.ts
import readline = require('readline')

readline
  .createInterface({
    input: process.stdin,
    output: process.stdout
  })
  .on('line', (line: string) => {
    console.log(line.toLowerCase().includes('problem') ? 'yes' : 'no')
  })
  .on('close', () => {
    process.exit(0)
  })
